docs/en/user_guides/classification.md

```diff
@@ -46,14 +45,23 @@ Remarks:
 
 ## Linear Evaluation
 
-The linear evaluation is one of the most general benchmarks, we integrate several papers' config settings, also including multi-head linear evaluation. We write classification model in our own codebase for the multi-head function, thus, to run linear evaluation, we still use `.sh` script to launch training. The supported datasets are **ImageNet**, **Places205** and **iNaturalist18**.
+The linear evaluation is one of the most general benchmarks, we integrate several papers' config settings, also including multi-head linear evaluation. We write a classification model in our own codebase for the multi-head function, thus, to run linear evaluation, we still use `.sh` script to launch training. The supported datasets are **ImageNet**, **Places205** and **iNaturalist18**.
```

Comment:
it is not only about linear evaluation, please update sub-title and description.

Reply:
This part is only about linear evaluation

Reply:
I mean that we need to revise this part, the linear eval and fine-tuning can be organized together

Reply:
done